### ISSUE TITLE
Messages for Presenters

### ISSUE THREAD

**Issue by kacollins** [techlahoma/help-wanted#87]
We could use help with creating a simple web app to display messages to presenters on the TV in the back of the room at StarSpace46.

Committee: OKC Broadcasting
Contact: Kimberly Collins, Aaron Krauss
Slack: kimberlycollins, thecodeboss, [#techlahoma-volunteers](https://techlahoma.slack.com/messages/techlahoma-volunteers)
Email: kcollins at techlahoma dot org

**Comment by kacollins** [comment]
@thecodeboss started a solution for this! https://github.com/alkrauss48/word-up

**Comment by thecodeboss** [comment]
@kacollins I think you tagged the wrong person, you must have meant to tag @alkrauss48

**Comment by kacollins** [comment]
Oops! That's what I get for copying and pasting from somewhere else. You'd think I would've noticed that the username in the link didn't match, but I didn't. Sorry about that!

**Comment by alkrauss48** [comment]
@thecodeboss yes - the infamous other thecodeboss! Perhaps one day we'll finally meet

**Comment by bshambaugh** [comment]
Hmm....it fits the screen width....then slowly expands vertically
